utilities/cli/README.md

```diff
@@ -265,6 +266,10 @@ Each mode is defined as a named object under the `modes` key:
       "production": {
         "description": "High-performance mode with GPU acceleration",
         "requirements": ["gpu", "model", "tensorrt"],
+        "env": {
+          "CUDA_VISIBLE_DEVICES": "0",
```

Comment:
Wouldn't we generally >=1 GPUs for runtime?

Is there a good way to either set env only for build time, or to "unset" a certain variable at runtime?

Reply:
thanks, updated the example, also added 'mode.build.env' following the same pattern, now support build time only vars.